sinonjs/sinon
# remove unnecessary properties quoting

#### Purpose

Remove unnecessary property quoting throughout the codebase except for places where this would lead to inconsistency (objects where some properties must be quoted). Also, this would unquote properties that are reserved words - according to the [specification, the "Reserved word usage" section](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Lexical_grammar) it is safe to use them as object properties - they just can't be used as variable names

Also, there is a stuff that been done by eslint in precommit hook - it has removed a lot of redundant trailing spaces and (probably) fix line endings in a few files. I can't commit without that changes so I hope they wouldn't be a problem

#### How to verify - mandatory
1. Check out this branch
2. `npm install`
3. `npm test-node`
4. `npm test-headless`

#### Checklist for author
